samples/openthread/cli/README.rst

```diff
@@ -518,6 +519,17 @@ To test the Thread 1.2 and Thread 1.3 features, complete the following steps:
             addresses: [fdde:ad00:beef:0:e0fc:dc28:1d12:8c2]
          Done
 
+Power consumption measurements
+==============================
+
+The Thread CLI sample can be used to perform power consumption measurements for Sleepy End Devices.
+After building and flashing with :file:`overlay-low_power.conf` and :file:`low_power.overlay` the device will start regular operation with UART console enabled.
+This allows for easy configuration of the device, specifically the SED polling period or the SSED CSL period, among other relevant parameters.
+Then, once the device becomes attached to a Thread Router it will automatically suspend UART operation and power down unused RAM.
+From this moment it is not possible to use the CLI to control the device, but it will periodically wake up from deep sleep mode and turn on the radio to receive any messages from its parent.
+
+If the device is connected to a Power Profiler Kit II, detailed power comsumption measurmens can be performed.
```

Comment:
```suggestion
If the device is connected to a Power Profiler Kit II, you can perform detailed power consumption measurements.
```

Reply:
Do you think we can add some appropriate PPK2 link here?

Reply:
Absolutely. You can use the \`Power Profiler Kit II (PPK2)\`_ syntax for that.